lnwallet/rpcwallet/healthcheck.go

```diff
@@ -11,14 +11,20 @@ import (
 // configuration.
 func HealthCheck(cfg *lncfg.RemoteSigner, timeout time.Duration) func() error {
 	return func() error {
-		_, err := connectRPC(
+		conn, err := connectRPC(
 			cfg.RPCHost, cfg.TLSCertPath, cfg.MacaroonPath, timeout,
 		)
 		if err != nil {
 			return fmt.Errorf("error connecting to the remote "+
 				"signing node through RPC: %v", err)
 		}
 
+		err = conn.Close()
```

Comment:
👍 

Stylistically, could choose to `defer` this, but doesn't make a hug difference either way.